### ISSUE TITLE
Each question/answer on new page

### ISSUE THREAD

**Issue by MDecker-MobileComputing** [MDecker-MobileComputing/KahootResult2Word#1]
Add a command line option `--newpage`, whose effect is it to write each question on its own page (i.e. newpage after each question).

This way it cannot happen that a give question will be "split", i.e. placed on two pages.

**Comment by MDecker-MobileComputing** [comment]
Info on how to implement page break with POI: 

https://stackoverflow.com/questions/28539760/apache-poi-page-breaks

Method `XWPFParagraph.setPageBreak(boolean)`

*"[...] the contents of this paragraph are rendered on the start of a new page in the document."*

**Comment by MDecker-MobileComputing** [comment]
Call method `setPageBreak(true)` on object `paragraph1` in method [`writeTrueFalseQuestion`](https://github.com/MDecker-MobileComputing/KahootResult2Word/blob/master/src/main/java/de/mide/kahoot/result2word/poi/KahootResultDocxWriter.java#L224) and on method `paragraph` in method [`writeMultiSingleChoiceQuestion`](https://github.com/MDecker-MobileComputing/KahootResult2Word/blob/master/src/main/java/de/mide/kahoot/result2word/poi/KahootResultDocxWriter.java#L287).

**Comment by MDecker-MobileComputing** [comment]
Changes needed in class [CmdLineArgsParser](https://github.com/MDecker-MobileComputing/KahootResult2Word/blob/master/src/main/java/de/mide/kahoot/result2word/utils/CmdLineArgsParser.java):

````
public static final String CMDLINE_OPTION_LETTER_N_FOR_NEWPAGE = "n";
````

New statement in `static{}` block:

````
Option helpOption = Option.builder(CMDLINE_OPTION_LETTER_N_FOR_HELP)
                                .required(false)
                                .longOpt("newpage")
                                .desc("Start new page for each question")
                                .hasArg(false)
                                .build();	
````

**Comment by MDecker-MobileComputing** [comment]
Changes in class [`Main`](https://github.com/MDecker-MobileComputing/KahootResult2Word/blob/master/src/main/java/de/mide/kahoot/result2word/Main.java):

Pass reference to object `cmdLine` (which is needed to query if a particular command line option is set or not) as argument when calling constructor of class `KahootResultDocxWriter` in [this line](https://github.com/MDecker-MobileComputing/KahootResult2Word/blob/master/src/main/java/de/mide/kahoot/result2word/Main.java#L251).

Passing the whole `cmdLine` object to the writer object is a footer-proof solution, because then also further command line options that might be added in the future can be queried in class `KahootResultDocxWriter` without the need to change the `Main` class.

**Comment by MDecker-MobileComputing** [comment]
Implemented as of [Commit ID `3dd279d213bf17281c17b74f8a0a35fa514e2acc`](https://github.com/MDecker-MobileComputing/KahootResult2Word/commit/3dd279d213bf17281c17b74f8a0a35fa514e2acc).
